lib/rucio/transfertool/fts3.py

```diff
@@ -92,7 +92,14 @@ def __init__(self, external_host, token=None):
         :param external_host:   The external host where the transfertool API is running
         :param token: optional parameter to pass user's JWT
         """
-        usercert = config_get('conveyor', 'usercert', False, None)
+
+        self.multi_vo = config_get_bool('common', 'multi_vo', False, None)
+        if self.multi_vo:
+            # a default cert that can be used incase of no selection for each VO that has basic permissions for the public methods
+            usercert = config_get('conveyor', 'usercert', False, None)
+        else:
+            # a cert that is used for the single VO
+            usercert = config_get('conveyor', 'usercert', False, None)
```

Comment:
I don't see any difference between if and else

Reply:
That is fair, I will remove the if/else

Reply:
I have removed the If/Else